api/pkg/db/migration/202107291608_refresh_all_tables.go

```diff
@@ -167,6 +235,12 @@ func migrateDB(txn *gorm.DB, log *log.Logger) error {
 			log.Error(err)
 			return err
 		}
+
+		query := fmt.Sprintf("ALTER SEQUENCE catalogs_id_seq RESTART WITH %d", catalogs_last_value+1)
```

Comment:
do we need to `+ 1` if we getting previous last value and restart with it back?

Reply:
Yes we have to because if there is a new entry then it considers the last value and inserts the value at the last value hence doing +1

Reply:
ah okay 👍🏽